adli_system.py

```
@@ -0,0 +1,72 @@
+        return -1
+
+    '''
+    Create a unique instance id that is passed to every program in the system.
+    The system definition file contains systemId and systemVersion which allow us
+    to identify the system. This unique id will be used to identify every deployment
+    of the system.
+    '''
+    instance_uid = str(uuid.uuid4())
+
+    for path in sys_def_file["programs"]:
+        output = subprocess.run(
```

Ruff (0.8.2)

Local variable `output` is assigned to but never used

Remove assignment to unused variable `output`

(F841)